Src/libCZI/CziParse.cpp

```diff
@@ -518,40 +518,76 @@ using namespace libCZI;
     throw std::logic_error("not (yet) implemented");
 }
 
-/*static*/void CCZIParse::AddEntryToSubBlockDirectory(const SubBlockDirectoryEntryDV* subBlkDirDE, const std::function<void(const CCziSubBlockDirectoryBase::SubBlkEntry&)>& addFunc)
+/*static*/void CCZIParse::AddEntryToSubBlockDirectory(const SubBlockDirectoryEntryDV* subBlkDirDV, const std::function<void(const CCziSubBlockDirectoryBase::SubBlkEntry&)>& addFunc, bool lax_subblock_coordinate_checks)
```

Comment:
Since the "lax" mode is actually the current behavior, wouldn't it make sense to rather introduce a "strict" mode (as we are adding additional optional checks to the current version)?
However, this is probably just a matter of taste, so please feel free to leave it as is if you like the "lax" point of view better!